.pre-commit-config.yaml

```diff
@@ -8,4 +8,4 @@ repos:
       language: system
       pass_filenames: false
       always_run: true
-      stages: [push]
\ No newline at end of file
+      stages: [pre-push]
```

Comment:
`pre-commit run --all-files` triggers `hook id `ci-checks` uses deprecated stage names (push) which will be removed in a future version.  run: `pre-commit migrate-config` to automatically fix this.`
running `pre-commit migrate-config` makes this modification to the pre-commit yaml